scenario/time_sync/result.rst

```diff
@@ -1,4 +1,118 @@
 Result
 ======
 
-TBD
+Setup environment
+^^^^^^^^^^^^^^^^^
+
+Install ROS environment
+-----------------------
+
+We have provided the way how to isntall ROS enviroment, please click `here <https://adlink-ros.github.io/roscube-doc/neuronsdk/neuron_library/installation.html#installation>`_.
```

Comment:
You have the wrong link here.
Should be https://adlink-ros.github.io/roscube-doc/neuronsdk/neuron_startup_menu/installation.html

Reply:
Oops, I will fix it.